chalice/deploy/models.py

```diff
@@ -165,17 +165,28 @@ def dependencies(self):
 
 
 @attrs
-class ScheduledEvent(ManagedModel):
-    resource_type = 'scheduled_event'
-    rule_name = attrib()            # type: str
-    schedule_expression = attrib()  # type: str
-    lambda_function = attrib()      # type: LambdaFunction
+class FunctionEventSubscriber(ManagedModel):
+    lambda_function = attrib()  # type: LambdaFunction
```

Comment:
If we want to refactor this, we will probably want to define the `dependencies()` method for this base class as well. It looks like it got dropped in the refactoring.